src/components/offer-card/offer-details/OfferDetails.jsx

```diff
@@ -0,0 +1,42 @@
+import { t } from 'i18next'
+
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import LanguageIcon from '@mui/icons-material/Language'
+
+import AppChip from '~/components/app-chip/AppChip'
+
+import { styles } from '~/components/offer-card/offer-details/OfferDetails.styles'
+
+const OfferDetails = ({ name, bio, subject, level, description, languages }) => {
+  return (
+    <Box sx={ styles.container }>
+      <Typography variant='h6'>{ name }</Typography>
+      <Typography sx={ styles.bio }>{ bio }</Typography>
+
+      <Box sx={ styles.chipsContainer }>
+        <AppChip
+          style={ styles.subjectChip }
+          labelStyle={ styles.subjectChipLabel }
+        >
+          { subject.toUpperCase() }
+        </AppChip>
+
+        <AppChip
+          style={ styles.levelChip }
+          labelStyle={ styles.levelChipLabel }
+        >
+          { `${t('common.beginner')} - ${level}`.toUpperCase() }
```

Comment:
if level is Beginer, we will get Beginer - Beginer?

Reply:
Done.